final_task/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include rss_reader/ARIALUNI.ttf
```

Comment:
This will not work on Windows OS, because is has different filesystem path separators (`\`)